src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html

```diff
@@ -349,12 +349,92 @@ <h3 *ngIf="displaySubsection(showQuestion, 4, 7)">Managing Session Responses</h3
+                  <label class="control-label">
+                    If showing only a section:
+                  </label>
+                  <div>
+                    <select class="form-control">
+                      <option [ngValue]="InstructorSessionResultSectionType.EITHER">
+                        Show response if either the giver or evaluee is in the selected section
+                      </option>
+                      <option [ngValue]="InstructorSessionResultSectionType.GIVER">
+                        Show response if the giver is in the selected section
+                      </option>
+                      <option [ngValue]="InstructorSessionResultSectionType.EVALUEE">
+                        Show response if the evaluee is in the selected section
+                      </option>
+                      <option [ngValue]="InstructorSessionResultSectionType.BOTH">
+                        Show response only if both are in the selected section
+                      </option>
+                    </select>
+                  </div>
+                </div>
+              </div>
+              <div class="col-md-6">
+                <label class="control-label">
+                  Additional settings:
+                </label>
+                <div ngbTooltip="Group results in the current view by team">
+                  <input type="checkbox"> Group by Teams
+                </div>
+                <div ngbTooltip="Show statistics">
+                  <input type="checkbox"> Show Statistics
+                </div>
+                <div ngbTooltip="Indicate missing responses">
+                  <input type="checkbox"> Indicate Missing Responses
+                </div>
+              </div>
+            </div>
+          </tm-example-box>
           <p>
             In the example below, results are sorted by <b>Giver > Recipient > Question</b>. Additionally, missing responses
             have been recorded.
           </p>
-          <p>EXAMPLE BOX</p>
+          <tm-example-box>
+            <tm-instructor-session-result-grq-view [session]="this.exampleFeedbackSession"
+                                                   [section]="'Section A'"
+                                                   [sectionType]="InstructorSessionResultSectionType.EITHER"
+                                                   [groupByTeam]="true"
+                                                   [showStatistics]="true"
+                                                   [indicateMissingResponses]="true"
+                                                   [isExpandAll]="true"
+                                                   [instructorCommentTableModel]="exampleInstructorCommentTableModel"
+                                                   [responses]="exampleGrqResponses"
+            ></tm-instructor-session-result-grq-view>
+            <tm-instructor-session-no-response-panel [session]="exampleFeedbackSession"
```

Comment:
This component have several buttons that throws an error in the console and cannot be easily disabled, outside of copy pasting the code. Should I do that instead?

Reply:
What errors? (Actually, it is always a good practice to report the error in details when seeking for help :P)

Reply:
Basically there's 2 buttons `Remind All` and `Submit response`.

Clicking on remind all does nothing and throws the following error to the console

<img width="1754" alt="image" src="https://user-images.githubusercontent.com/32454748/83709622-84468f80-a651-11ea-9515-73eb7af8c31c.png">

Clicking on `Submit response` will redirect them away to a non-working `Submit feedback` page and print an error on the console.

<img width="1772" alt="image" src="https://user-images.githubusercontent.com/32454748/83709808-dedfeb80-a651-11ea-92a9-632c8f549525.png">

Reply:
I think you can just disable those buttons.

However,

The first one is a bug and we should fix it. In normal case, there should be a modal pop out. Maybe @Derek-Hardy can help in this case as he has worked on the component before.

The second error is not a bug (try to figure out why?).

Reply:
I think the second error is an intended error since it is an invalid request.

So I'll just disable the buttons then.

Reply:
The first one might not be the bug, you may just forget to import the `XXModal`. Your failing CI also indicates that.

Reply:
But I don't think that we should enable the first button nonetheless, since this is just an example component?

Reply:
Yup. We can just disable it.

Reply:
Okay!